tools/oot-driver/playbooks/roles/deploy_sro/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: Create the SRO namespace
+  kubernetes.core.k8s:
+    api_version: v1
+    kind: Namespace
+    name: "{{ sro_namespace }}"
+    kubeconfig: "{{ kubeconfig }}"
+    state: present
+
+- name: Create the SRO operator group
+  kubernetes.core.k8s:
+    api_version: operators.coreos.com/v1
+    kind: OperatorGroup
+    name: openshift-special-resource-operators
+    namespace: "{{ sro_namespace }}"
+    kubeconfig: "{{ kubeconfig }}"
+    state: present
+
+- name: Create the SRO subscription
+  kubernetes.core.k8s:
+    definition: "{{ lookup('file', '../../../resources/sro_subscription.yml') | from_yaml }}"
```

Comment:
can't the subscription be also in place here? there is a reason to have it different from the OperatorGroup?

Reply:
The reason why I put it as a file instead of using the kubernetes.core.k8s module is I was not sure how to pass the metadata section of the YAML file through that module.